python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -65,7 +65,7 @@ cdef class KafkaDatasource(Datasource):
 
     cpdef int64_t get_committed_offset(self, string topic, int32_t partition)
 
-    cpdef map[string, vector[int32_t]] list_topics(self, string specific_topic)
+    cpdef map[string, vector[int32_t]] list_topics(self, string topic) except +
```

Comment:
Since this is a Python method that calls a C++ function, it can raise both Python and C++ exceptions. We should mark this one `except +*`.

Although, it's also OK to just mark it `except *`, since the C++ exception will already have been converted to a Python exception because we marked the `list_topics` C++ function `except +`.

I hope I'm making sense :)